packages/fast-color-explorer/app/app.tsx

```diff
@@ -0,0 +1,119 @@
+                                    <AutoSizer
+                                        onResize={
+                                            /* this lambda is intentional - it forces the pure component to re-render */ (): void => {}
+                                        }
+                                    >
+                                        {this.renderColorBlockList}
+                                    </AutoSizer>
+                                </Row>
+                                <Row>
+                                    <Gradient
+                                        colors={palette(PaletteType.neutral)(
+                                            this.props.designSystem
+                                        )}
+                                        createAnchors={true}
+                                        scrollToItem={this.handleGradientScroll}
+                                    />
+                                </Row>
+                                <Row>
+                                    <Gradient
+                                        colors={palette(PaletteType.accent)(
+                                            this.props.designSystem
+                                        )}
+                                        createAnchors={false}
+                                    />
+                                </Row>
+                            </Container>
+                        </Canvas>
+                        <Background value={DarkModeBackgrounds.L4}>
+                            <ControlPane />
+                        </Background>
+                    </Row>
+                </Container>
+            </DesignSystemProvider>
+        );
+    }
+
+    private handleGradientScroll = (index: number, align: string): void => {
+        if (this.colorBlockScrollerRef.current !== null) {
+            this.colorBlockScrollerRef.current.scrollToItem(index, align);
+        }
+    };
+
+    private renderColorBlockList = (props: any): JSX.Element => {
+        return (
+            <FixedSizeList
+                width={props.width}
+                height={props.height}
+                itemSize={400}
+                layout={"horizontal"}
+                itemCount={this.props.designSystem.neutralPalette.length}
+                itemKey={(index: number): string =>
+                    palette(PaletteType.neutral)(this.props.designSystem)[index]
+                }
+                ref={this.colorBlockScrollerRef}
+            >
+                {this.renderColorBlock}
+            </FixedSizeList>
+        );
+    };
+
+    private renderColorBlock = (props: any): JSX.Element => {
+        const color: string = palette(PaletteType.neutral)(this.props.designSystem)[
+            props.index
+        ];
+
+        return (
+            <div style={props.style} key={color}>
+                <ColorBlocks {...{ backgroundColor: color, index: props.index } as any} />
```

Comment:
shouldn't this just be `<ColorBlocks backgroundColor={color} index={props.index} />`?

Reply:
yea but react-redux is doing somthing funny with the typings and that errors.